telegram-bot/main.go

```diff
@@ -168,12 +191,15 @@ func main() {
 	}
 
 	fmt.Println("Connected to MongoDB!")
+	//tweet
```

Comment:
Remove this comment or add a better one here, this doesn't add any value for now.